price_service/client/js/package.json

```diff
@@ -50,7 +50,7 @@
   "dependencies": {
     "@pythnetwork/price-service-sdk": "*",
     "@types/ws": "^8.5.3",
-    "axios": "^1.2.5",
```

Comment:
I downgraded axios and it seemed to fix the problem

Reply:
lol what... nice find though